neurovault/urls.py

```diff
@@ -380,6 +389,7 @@ class CollectionViewSet(mixins.RetrieveModelMixin,
     filter_fields = ('name', 'DOI', 'owner')
     serializer_class = CollectionSerializer
     filter_backends = (DjangoFilterBackend,)
+    permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
```

Comment:
Since there is not a clear way to map addition of related object to django object permissions I think your solution is adequate. Hopefully the object level permission will get handy when we start modifying or deleting objects.